### ISSUE TITLE
fresh install does not include config.hpp and exports.hpp

### ISSUE THREAD

**Issue by pgbarletta** [chemfiles/chemfiles#60]
I recently cloned from github repo to **~/chemfiles** and followed installation instructions, using these options for the cmake command:
```
 cmake -DCMAKE_BUILD_TYPE=debug -DCHFL_BUILD_TESTS=ON -DCHFL_ENABLE_NETCDF=ON ..
```
But for some reason both these files weren't present in the **~/chemfiles/include/chemfiles** folder, nor in the installed folder **/usr/local/include/chemfiles**, so when I tried to compile my code (including the flag **-I/usr/local/include**), the following error appeared:

```
/usr/local/include/chemfiles.hpp:11:32: fatal error: chemfiles/config.hpp: No such file or directory
```
The error was fixed by copying these 2 files to **/usr/local/include/chemfiles**.

**Comment by Luthaf** [comment]
Are you sure you are at the latest master? This should have been fixed by 5c6ac94445e78adfd74a1005de64001a0949bf17. 

If you are following the master for your own code, remember to do a `git pull && git submodule update` every few weeks. Else, I should release the 0.7.0 version in the next weeks, so that you can just stay at this version.

**Comment by pgbarletta** [comment]
Oh, good. It's already solved.
